### ISSUE TITLE
Remove support for deprecated cinder APIv1

### ISSUE THREAD

**Issue by kamidzi** [bloomberg/chef-bcpc#802]
As per warning:
```
2015-08-28 15:48:32.996 24588 WARNING cinder.api [-] The v1 api is deprecated and will be removed in the Liberty release. You should set enable_v1_api=false and enable_v2_api=true in your cinder.conf file.
```

**Comment by erhudy** [comment]
I ran into issues when attempting to removing the v1 API initially (hence why there are both **volume** and **volumev2** endpoints in the service catalog). Something wasn't happy when I took v1 away, and other things weren't happy when I took v2 away, so I ended up turning them both on. I don't remember what was unhappy now, though.

**Comment by erhudy** [comment]
Liberty fully deprecates v1. According to the release notes:
```
Horizon requires both a volume and volumev2 endpoint for Cinder, even if only using v2.
```
So for Liberty, we could switch to running only v2 and then do that hack in the service catalog.

**Comment by erhudy** [comment]
This is fixed in #1000, which disables Cinder API v1 and modifies the service catalog to point both the v1 and v2 endpoints at v2.

**Comment by kamidzi** [comment]
As per previous comment.
